source/simulator/parameters.cc

```diff
@@ -1004,6 +1004,12 @@ namespace aspect
                          "advected with the melt velocity instead of the solid velocity. "
                          "This method can only be chosen if melt transport is active in the "
                          "model."
+                         "\n"
+                         "\\item ``prescribed field'': If a compositional field is marked "
+                         "with this method, then the value of a specific additional material "
```

Comment:
can you add at the beginning that the value of these fields is determined in each timestep from the material model?